core/services/ocr2/plugins/s4/plugin.go

```diff
@@ -2,15 +2,31 @@ package s4
 
 import (
 	"context"
+	"math/big"
 	"time"
 
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/smartcontractkit/libocr/commontypes"
```

Comment:
What are these imports needed for?

Reply:
🤔 not sure why `/smartcontractkit/libocr/commontypes` is marked as a new addition, this is already in [dev](https://github.com/smartcontractkit/chainlink/blob/306eadcf505b54502b7112efc395457d4bd0f3c5/core/services/ocr2/plugins/s4/plugin.go#L11) and is used by the logger